packages/transformer/src/IModelImporter.ts

```diff
@@ -98,6 +103,28 @@ export class IModelImporter implements Required<IModelImportOptions> {
 
   private static _realityDataSourceLinkPartitionStaticId: Id64String = "0xe";
 
+  private _duplicateCodeValueMap?: Map<Id64String, string>;
+
+  /**
+   * A map of conflicting element code values.
+   * In cases where updating an element's code value results in a codeValue conflict, the element's code value will instead be updated to a random guid.
+   * The actual codeValue will be stored in this ElementId->CodeValue map.
+   * This is needed in cases where the duplicate target element is set to be deleted in the future, or when elements are switching code values with one another and we need a temporary code value.
+   * To resolve code values to their intended values call [[IModelImporter.resolveDuplicateCodeValues]].
+   *
+   * @returns undefined if no duplicates were detected.
+   */
+  public get duplicateCodeValueMap(): Map<Id64String, string> | undefined {
```

Comment:
I would refrain from making this public. See comment on the `catch` below
(same with the getters/setters which I also think are unnecessary)

Reply:
changed everything to private